packages/common/src/api/fetcher.ts

```
@@ -1,6 +1,10 @@
 import axios, {AxiosRequestConfig, AxiosResponse} from 'axios'
+
 import {ApiError, NetworkError} from '../errors/errors'
 
+/**
+ * @deprecated This function is deprecated and will be removed in a future release. Use `fetchData` instead.
```

marked as deprecated.

---

i am using it in the resolver package 👍🏻

---

@banklesss definitely not a issue for now, the APIError that is throw by it is ignored at the root level of the API. I think we wont display any error to the user if the resolver request can’t go through (fail silently)